autoptsserver.py

```diff
@@ -68,6 +68,7 @@ def _com_port_exists(port_name):
     """Check if the COM port exists."""
     ports = serial.tools.list_ports.comports()
     for port in ports:
+        print(f"Port_device: {port.device}, port_name: {port_name}")
```

Comment:
Since `_com_port_exists` is called once per port per dongle, then we end up printing this n * m times, which seems very verbose